### PR TITLE
Add more rules to rhel9 stig

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:

Add more rules to the RHEL 9 STIG.

#### Rationale:
To keep in sync with development.

### CODE REVIEW

File: linux_os/guide/system/software/integrity/crypto/configure_libreswan_crypto_policy/rule.yml [hunk 1]
```diff
@@ -47,6 +47,9 @@ ocil: |-
 
 fixtext: |-
     Configure Libreswan to use the system cryptographic policy.
+
+    If the ipsec service is not installed is not applicable.
```
Comment (ggbecker) [inline linux_os/guide/system/software/integrity/crypto/configure_libreswan_crypto_policy/rule.yml:51]:
```suggestion
    If the IPSec service is not installed, this check is Not Applicable.
```

Comment (ggbecker) [reply linux_os/guide/system/software/integrity/crypto/configure_libreswan_crypto_policy/rule.yml:51]: or maybe this piece of information should be in OCIL clause instead. In RHEL8 STIG this usually goes the check content part.

File: controls/srg_gpos/SRG-OS-000342-GPOS-00133.yml [hunk 2]
```diff
@@ -11,4 +11,5 @@ controls:
             - rsyslog_encrypt_offload_actionsendstreamdrivermode
             - rsyslog_encrypt_offload_defaultnetstreamdriver
             - rsyslog_remote_loghost
+            - auditd_audispd_configure_sufficiently_large_partition
```
Comment (jan-cerny) [inline controls/srg_gpos/SRG-OS-000342-GPOS-00133.yml:14]: This rule has been removed previously based on a discussion in https://github.com/ComplianceAsCode/content/pull/8924. What changed or why do we add it back?

Comment (Mab879) [reply controls/srg_gpos/SRG-OS-000342-GPOS-00133.yml:14]: We might need a new rule for the audited version of this. The idea is valid, but the check is not.

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**